TimberWinR/mdocs/Generator.md

```diff
@@ -0,0 +1,37 @@
+# Input: Generator
+
+The Generator input can be used to Generate log files for test purposes.
+
+## Parameters
+The following parameters are allowed when configuring WindowsEvents.
```

Comment:
Sorry if I wasn't clear - I think you might have a copy paste error in the Parameters description line. I think "configuring WindowsEvents" should read "configuring a Generator input".